### PR TITLE
Allow styling of outer container of SearchBar

### PR DESCRIPTION
**Pull request by ceckobecko**
Also swap the names so now the containerStyle
corresponds to the SearchBar container
and introduce the new inputContainerStyle to
match the text input container.

Update docs

### CODE REVIEW

**Comment by xavier-villelegier** [comment]
Hey @ceckobecko, thanks for your help to improve the `v1.0.0` 🙏

Could you add some tests on this new props and regenerate the snapshots ?

We are currently working on the `v1.0.0-beta3` which will "only" contain React Native Web support. Your PR will be merged for the `v1.0.0-beta4` 👍

**Comment by ceckobecko** [comment]
Sure! Will add some tests.

**Comment by ceckobecko** [comment]
Also improved isEmpty state to properly be set if initial value is different from empty string

File: src/searchbar/SearchBar-android.js [hunk 1]
```diff
@@ -54,14 +54,15 @@ class SearchBar extends Component {
     super(props);
     this.state = {
       hasFocus: false,
-      isEmpty: true,
+      isEmpty: !props.value,
```
Comment (xavier-villelegier) [inline src/searchbar/SearchBar-android.js]: Nice catch ! Maybe also check for `defaultValue` which is a react native `TextInput` props

**Comment by ceckobecko** [comment]
Also allow getting a reference to the actual search bar.

**Comment by iRoachie** [comment]
Hey @ceckobecko since next has been restored(removed beta3 and react-native-web). I think you have to do over this PR or rebase from next

**Comment by ceckobecko** [comment]
Sure thing! Will rebase it from `next`

**Comment by xavier-villelegier** [comment]
Hey @ceckobecko, you still have all React Native Web support on your PR, I think it's easier to open a new PR, maybe you can cherry-pick your commits if you don't want to start again from scratch. Sorry about that 🙏

**Comment by ceckobecko** [comment]
Closing this in favor of #938

**Pull request closed without merging**